content/workshops/openshift_4_101_dynatrace/fin.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Fin.
-workshops: openshift_4_101
+workshops: openshift_4_101_dynatrace
 workshop_weight: 20
```

Comment:
Should be changed to "workshop_weight: 23". Otherwise "fin" shows up between lab9 and lab 10

Reply:
Fixed; thanks!